### DISCUSSION TITLE
v0.10.0

### DISCUSSION THREAD

**Discussion by Hayao-H** [Hayao-H/Niconicome#431] [一般]
# 概要
## 追加
- スナックバー表示時間の設定を追加（**「一般設定」**） #425
## 修正
- すべての過去ログをDL出来ない問題を修正 #313
 **「ダウンロード設定」から「試験的なコメント...」を有効にする必要があります。**
- コメント数が多い動画でコメントの取得に失敗する問題を修正  #426
  **「ダウンロード設定」で「過去ログ取得時の待機時間」を適宜設定してください。（1500ms程度？）**

<hr /><em>This discussion was created from the release <a href='https://github.com/Hayao-H/Niconicome/releases/tag/v0.10.0'>v0.10.0</a>.</em>

**Comment by UmiAkai** [comment]
初めまして、0.10.0から使えてます。
チャンネルの暗号化動画がリストできません
例えば「https://ch.nicovideo.jp/k-on_anime」お追加すると、一話以外リストできません、症状は：
[視聴ページ解析プラグイン]視聴ページの解析に失敗しました。(詳細:Jsonの解析に失敗しました。, 例外:Unexpected end of JSON input)
so34345824の取得に失敗しました。(詳細:視聴ページの解析に失敗しました。(サーバーエラー・プラグインでエラー))

単独のIDも同じく、他のチャンネルもそうです。ログインはFirefoxで、一話は無事コメDLできます、そしてFirefoxで開けると情報やコメントや過去コメなど普通に見えます。

何か設定どかの問題か、宜しくお願い致します。

**Reply by UmiAkai** [reply]
cookies公有ログインの問題みたい。パスワードログインにしたらDL成功しました

**Reply by Hayao-H** [reply]
ご報告ありがとうございます。また、リプライが遅れてしまい申し訳ありません。
問題は現在も発生していますか？
